cb_hr_views/models/hr_employee.py

```diff
@@ -222,8 +222,15 @@ def action_open_related_partner(self):
 
     @api.multi
     def _compute_show_leaves(self):
-        for employee in self:
-            employee.show_leaves = employee.show_info
+        for rec in self:
+            rec.show_leaves = rec.show_info and rec.is_employee
```

Comment:
Deberiamos añadir show_info y is_employee al api.depends()?

Reply:
ufff, well, as show_info is computed all the relateds of show_info, isn't it?

Reply:
well, if it works this way lets keep it like this